src/%IPM/Main.cls

```diff
@@ -2383,11 +2408,208 @@ ClassMethod UpdateLanguageExtensions(pVerbose As %Boolean = 0, pTestOnly As %Boo
 	Quit tSC
 #; These are the actual contents of %ZLANGC00 (to be added/updated)
 $$$STARTTAG
-ZPM(pArgs...) If $System.CLS.IsMthd("%IPM.Main", "Shell") { Do ##class(%IPM.Main).Shell(pArgs...) Quit } Else { Write !, "InterSystems Package Manager is not installed in this namespace." Quit }
+ZPM(pArgs...)
+ If $System.CLS.IsMthd("%IPM.Main", "Shell") { 
+ 	Do ##class(%IPM.Main).Shell(pArgs...) Quit
+ } Else {
+	// TODO: Decide what to do here; definitely needs to support enabling IPM from here as well,
+	// but need to decide what level of customization to provide
+	Quit
+ }
 $$$ENDTAG
 #; Need an extra line down here to avoid the end bracket being on the same line as $$$ENDTAG - a comment is just fine.
 }
 
+ClassMethod EnableIPM(ByRef pCommandInfo)
+{
+	Write !, "Version of IPM in current namespace: "
+	Do ..GetListModules("zpm",.list)
```

Comment:
Nit: Perhaps put the name of the module in a macro (and use the same macro in the `Version()` method of this class